src/S3SparseIterator.cpp

```diff
@@ -154,7 +154,6 @@ void S3SparseIterator::pushSamples(std::ostringstream* oss) {
     for (uint64_t j = 0; j < minibatch_rows; ++j) {
       if (use_label) {
         FEATURE_TYPE label = load_value<FEATURE_TYPE>(s3_data); // read label
-        assert(label == 0.0 || label == 1.0);
```

Comment:
I would like to leave some code here to check the labels (inside a #debug pragma).